taskvine/src/manager/vine_manager_get.c

```diff
@@ -355,6 +356,134 @@ vine_result_code_t vine_manager_get_single_file(struct vine_manager *q, struct v
 	return vine_manager_get_buffer(q, w, 0, f, &total_bytes);
 }
 
+
```

Comment:
Can we simplify fetching of stdout to just do a normal `get` in the usual way?
-